oidc_apis/scopes.py

```diff
@@ -97,6 +101,44 @@ def scope_address(self):
         return super().scope_address()
 
 
+class SuomiFiUserAttributeScopeClaimsMeta(type):
+    def __dir__(cls):
+        names = super().__dir__()
+        for level in SuomiFiAccessLevel.objects.all():
+            names.append('info_suomifi_' + level.shorthand)
+        return names
+
+    def __getattr__(cls, name):
+        if name.startswith('info_suomifi_'):
+            try:
+                level = SuomiFiAccessLevel.objects.get(shorthand=name[13:])
```

Comment:
I understood where this 13 comes from, but still...
How about

  SUOMIFI_ATTR_PREFIX = re.compile('^info_suomifi_')

and then

  shorthand=re.sub(SUOMIFI_ATTR_PREFIX, '', name)

Or something like that?

Reply:
Or use the regular expression for both, matching the prefix and picking the correct substring:

```python
        match = re.match(r'^info_suomifi_(.*)', name)
        if match:
            try:
                level = SuomiFiAccessLevel.objects.get(shorthand=match.group(1))
```

Reply:
Updated.